src/components/layout/Workflow/Workflow.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+
+import { useGlobalState } from '../../../state/contexts/GlobalState';
+import { useRegistrationState } from '../../../state/contexts/RegistrationState';
+import { WorkflowProps } from '../../../types';
+import WorkflowButtons from '../../inputs/buttons/WorkflowButtons/WorkflowButtons';
+
+function Workflow({ stages }: WorkflowProps) {
+  const [{ stage, isFirstStage, isLastStage }, dispatchRegisterState] =
+    useRegistrationState();
+  const [{ walletAddress, jwk, isSearching }, dispatchGlobalState] =
+    useGlobalState();
+
+  const [nextCondition, setNextCondition] = useState<boolean>(false);
+  const [backCondition, setBackCondition] = useState<boolean>(false);
+
+  useEffect(() => {
+    // eslint-disable-next-line
```

Comment:
did this complain?

Reply:
yes, no-unused-vars on the key from object.entries

Reply:
FWIW I think you can set it an empty array to avoid this